### PR TITLE
Added new icon on Home Page

### LINKED ISSUE #60: Add new JSLovers logo in the website
**Is your feature request related to a problem? Please describe.**
Add new JSLovers logo to the website from here: https://jslovers.com/

**Describe the solution you'd like**
Add new JSLovers logo to the website

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by akshat99812**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #60 

**Snapshots/Videos:**
Screenshot 2024-06-12 at 1 56 34 PM

### CODE REVIEW

File: components/CarouselSection.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ const CarouselSection = ({
   const [api, setApi] = useState<CarouselApi>();
   const [current, setCurrent] = useState(0);
   const [count, setCount] = useState(0);
+  const [isClient,setIsClient]=useState(false)
```
Comment (Neha) [inline components/CarouselSection.tsx:32]: any reason for changes in this file?

File: components/CarouselSection.tsx [hunk 2]
```diff
@@ -49,13 +50,20 @@ const CarouselSection = ({
     return <>{gridChildren}</>;
   }
 
+  useEffect(()=>{
```
Comment (Neha) [inline components/CarouselSection.tsx:53]: same as above

File: components/Icons.tsx [hunk 3]
```diff
@@ -2,38 +2,12 @@
 
 export const Icons = {
   logo: () => (
-    <svg
+    <img
```
Comment (Neha) [inline components/Icons.tsx:5]: Can you please make alt tag "JSLovers" . Also, do not use the live website path. you should download image , save in the resources or assets folder of this project and link to src.

File: package-lock.json [hunk 4]
```diff
@@ -326,6 +326,126 @@
         "node": ">= 10"
       }
     },
+    "node_modules/@next/swc-darwin-x64": {
```
Comment (Neha) [inline package-lock.json:329]: why these changes are here?

**Review by Neha — COMMENTED** [review]
Thank you for your contribution. I have left a few comments please review

**Comment by Neha** [comment]
nitpick: the PR name should be "Added new logo on Home Page" not added new "icon"

File: components/CarouselSection.tsx (hunk 1 above)
Comment (akshat99812) [reply components/CarouselSection.tsx:32]: Sry ,was trying to fix the issue#53 About us section showing Hyradtion error

File: components/Icons.tsx (hunk 3 above)
Comment (akshat99812) [reply components/Icons.tsx:5]: Thank you for the feedback! 

I will update the alt tag to "JSLovers" and download the image to save it in the resources or assets folder of the project, linking it to the src accordingly. I'll also rename the file to logo as suggested.

Thanks again for the guidance!

**Comment by Neha** [comment]
Do you want to keep this PR?  Please avoid adding new PR of the same issue.
